### PR TITLE
DI-258 Monitor DLQ Failures

### PR DESCRIPTION
**Pull request by rohith1122**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-258

## Description

Track how many Changed Events and Change Requests are being directed into the 2 DLQs within the DI Pathway in order to see at a glance whether there are any potential bottlenecks and/or connectivity failures.

### Noteworthy Changes

- Added new DLQ Failures graph to the cloud watch dashboard

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing

Please tick the testing that has been completed

- [x] Unit tests
- [ ] Integration tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Integration tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

File: application/fifo_dlq_handler/fifo_dlq_handler.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from common.dynamodb import add_change_request_to_dynamodb
 from common.middlewares import set_correlation_id, unhandled_exception_logging
 from common.utilities import extract_body, get_sequence_number
+from eventbridge_dlq_handler.eventbridge_dlq_handler import handle_sqs_msg_attributes
```
Comment (ed-sparkes) [inline application/fifo_dlq_handler/fifo_dlq_handler.py:7]: could this move to common rather than having one dlq reference another

Comment (rohith1122) [reply application/fifo_dlq_handler/fifo_dlq_handler.py:7]: I tried in the previous commit but I struggled with sharing the fixture @dead_letter_message.

**Review by ed-sparkes — APPROVED** [review]

File: application/fifo_dlq_handler/fifo_dlq_handler.py (hunk 1 above)
Comment (mabe13) [reply application/fifo_dlq_handler/fifo_dlq_handler.py:7]: Had a look over the PR I think this is still an issue the `handle_sqs_msg_attributes` function is define in `common.utilities` but it being imported via the eventbridge dlq handle

**Review by mabe13 — APPROVED** [review]
LGTM 👍

**Review by mabe13 — APPROVED** [review]

**Review by ed-sparkes — APPROVED** [review]

**Review by mabe13 — APPROVED** [review]

**Pull request merged**